cabot/metricsapp/api/elastic.py

```diff
@@ -1,11 +1,54 @@
 from elasticsearch import Elasticsearch
+from enum import Enum
+
+
+class SupportedMetrics(Enum):
+    # Response includes the field "value"
+    METRICS_SINGLE = ['min', 'max', 'avg', 'value_count', 'sum', 'cardinality', 'moving_avg', 'derivative']
```

Comment:
Also, looking at your usage pattern, you should probably use `set()` instead of `list`

Also predefine
`METRICS_ALL = METRICS_SINGLE.union(METRICS_MULTIPLE)`